Create ui-c8y-1022-17-0-use-new-datapoint-explorer-component-for-device-page-measurements-tab.md (#3636)
SoftwareAG/c8y-docs: content/change-logs/device-management/ui-c8y-1022-17-0-use-new-datapoint-explorer-component-for-device-page-measurements-tab.md

---
date: ""

> title: Preview of the new Measurements tab based on the data explorer

Reviewer: @bozhidardedov I assume this refers to the data explorer and not a "data point explorer" as it said previously?

Reviewer 2: I guess the terms are used interchangeably. Data point explorer is used in implementation, UI mostly uses Data explorer.
<img width="655" height="609" alt="image" src="https://github.com/user-attachments/assets/adb7289b-da02-4df9-990b-57a7b03573be" />

Reviewer: We should use the UI terms in the user documentation. So it is ok as it is.

> title: Use new DatapointExplorerComponent for device page Measurements tab

Reviewer 2: ```suggestion
title: Feature preview of the new Measurements tab based on the Data point explorer
```

> title: Feature preview of the new Measurements tab based on the Data point explorer

Reviewer: ```suggestion
title: Preview of the new Measurements tab based on the data explorer
```

product_area: Device management & connectivity
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-4839
version: 1022.17.0
---

> Co-authored-by: Bozhidar Dedov <95280800+bozhidardedov@users.noreply.github.com>

Reviewer 2: ```suggestion
To provide a more powerful and flexible way to explore device measurements, the **Measurements** tab on the device page has been updated to use the new Data point explorer. This change allows users to interactively analyze device data in a more intuitive and efficient manner compared to the previous implementation. The new **Measurements** replaces the old one when the **Data point explorer** feature preview toggle is turned on.
```

> To provide a more powerful and flexible way to explore device measurements, the **Measurements** tab on the device page has been updated to use the new Data point explorer. This change allows users to interactively analyze device data in a more intuitive and efficient manner compared to the previous implementation. The new **Measurements** replaces the old one when the **Data point explorer** feature preview toggle is turned on.

Reviewer: ```suggestion
To provide a more powerful and flexible way to explore device measurements, the **Measurements** tab in the device details has been updated to use the new data explorer. This change allows users to interactively analyze device data more intuitively and efficiently. The new **Measurements** tab replaces the old one if the **Data explorer** feature preview toggle is enabled.
```
